client/src/components/FileBrowser.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <v-card
+    height="100%"
+    outlined
+  >
+    <v-card-title>
+      <router-link
+        :to="{ name: 'focus', query: { location: rootDirectory } }"
+        style="text-decoration: none;"
+        class="mx-2"
+      >
+        {{ '.' }}
+      </router-link>
+      <template v-for="(part, i) in splitLocation">
+        /
+        <template v-if="part">
+          <router-link
+            :key="part"
+            :to="{ name: 'focus', query: { location: locationSlice(i) } }"
+            style="text-decoration: none;"
+            class="mx-2"
+          >
```

Comment:
I think either both of these router links need to be changed to `path: '/focus'`, or the route for `Focus.vue` in the router needs to be updated with a name. Currently, clicking these links fail.

Reply:
It also seems like clicking on these segments doesn't actually update the file list, I suspect because that change isn't being watched for. FWIW, I don't know if we really even need these clickable file segments, we could probably just get away with a path and the `..` for navigation, if we really needed to.